cmd/operator-sdk/run/local.go

```diff
@@ -75,7 +76,11 @@ func (c runLocalArgs) runGo() error {
 	projutil.MustInProjectRoot()
 	absProjectPath := projutil.MustGetwd()
 	projectName := filepath.Base(absProjectPath)
-	outputBinName := filepath.Join(scaffold.BuildBinDir, projectName+"-local")
+	var exeSuffix string
+	if runtime.GOOS == "windows" {
+		exeSuffix = ".exe"
+	}
+	outputBinName := filepath.Join(scaffold.BuildBinDir, projectName+"-local"+exeSuffix)
```

Comment:
Just a nit to remove the need for the `exeSuffix` variable:

```
outputBinName := filepath.Join(scaffold.BuildBinDir, projectName+"-local")
if runtime.GOOS == "windows" {
	outputBinName += ".exe"
}

Reply:
Hi @al45tair, 

Could you please check the above nit? I think it is missing just this detail.

Reply:
I'll update the pull request tomorrow. Amusingly I originally wrote it the way @joelanford suggests, but changed my mind because it seemed cleaner to build the complete path element and then `filepath.Join()` *that*. But you're right, the cost is that we need a variable.

Reply:
I've updated the pull request, but there's a test failure that I don't think is anything to do with this change.

Reply:
I just restarted. We need to try to upgrade the marker to see if this problem is solved. It has been failing somethings and the current workaround is just to re-start it.